USER_GUIDE.md

```diff
@@ -219,6 +220,21 @@ var config = new ConnectionSettings(endpoint, connection);
 var client = new OpenSearchClient(config);
 ```
 
+### Amazon OpenSearch Serverless
+To configure signing when making requests to [Amazon OpenSearch Serverless](https://aws.amazon.com/opensearch-service/features/serverless/) is nearly identical to all above configuration for AwsSigV4, the only difference being the need to configure the service identifier.
```

Comment:
Nit: Maybe not so english :) How about "Use the `oass` service identifier to make requests to ..."?